web-services-app/pom.xml

```diff
@@ -161,6 +161,12 @@
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock-jre8</artifactId>
+            <version>${wiremock.version}</version>
```

Comment:
We don't need to specify the version here since it's already set in the main pom.xml